packages/frontend/src/components/Button/style.module.scss

```diff
@@ -1,9 +1,9 @@
 .button {
   align-items: center;
-  background-color: transparent;
+  background-color: var(--buttonSecondaryBackground);
   border-radius: 10px;
-  border: 0;
-  color: var(--buttonText);
+  color: var(--buttonSecondaryText);
+  border: 1px solid #ccc;
```

Comment:
Also take a look a proxy. I think we need a margin there.

![image](https://github.com/user-attachments/assets/5a268a09-2ea7-4802-98b5-6670cbf9fd80)

Reply:
Since the proxy component was introduced later than this PR it was not adapted. I applied the new borlderless styling now where needed